<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FDesktopRecorder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButtonEntirescreen">
        <property name="toolTip">
         <string>When the recording starts, it will record your entire desktop</string>
        </property>
        <property name="statusTip">
         <string>Records your entire desktop</string>
        </property>
        <property name="text">
         <string>Entire screen</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButtonSinglewindow">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string extracomment="dsfdsfds">When you start the recording, you will have to choose the window that you want to record</string>
        </property>
        <property name="statusTip">
         <string>Record a single window or screen.</string>
        </property>
        <property name="text">
         <string>Single Window</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButtonStartrecord">
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Start a recording</string>
        </property>
        <property name="text">
         <string>Record</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/recording.png</normaloff>:/images/recording.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pushButtonStoprecord">
        <property name="statusTip">
         <string>Stop the recording</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBoxRecordaudio">
        <property name="toolTip">
         <string>If checked, it will record no audio from the microphone</string>
        </property>
        <property name="statusTip">
         <string>Mute the microphone</string>
        </property>
        <property name="text">
         <string>Mute microphone</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbout"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionOpen_recording_directory"/>
   <addaction name="actionConsole"/>
   <addaction name="separator"/>
   <addaction name="actionPreviewrecording"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEditConsole">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Shows information about this program</string>
   </property>
   <property name="statusTip">
    <string>About FDesktopRecorder</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Let you change an amount of different settings like recording framerate etc.</string>
   </property>
   <property name="statusTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>Console</string>
   </property>
   <property name="toolTip">
    <string>Shows a console where you can see the output. Usefull if recording fails.</string>
   </property>
   <property name="statusTip">
    <string>Show/Hide console output</string>
   </property>
  </action>
  <action name="actionOpen_recording_directory">
   <property name="text">
    <string>Open recording directory</string>
   </property>
   <property name="toolTip">
    <string>Open directory containing your recordings</string>
   </property>
   <property name="statusTip">
    <string>Open directory containing your recordings</string>
   </property>
  </action>
  <action name="actionPreviewrecording">
   <property name="text">
    <string>Preview recording</string>
   </property>
   <property name="toolTip">
    <string>Preview the recording</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
